### PR TITLE
Data Atlas: Integration of vector data

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[[TERRANOVA] 12. Data Atlas: Vektordaten integrieren](https://trello.com/c/XheBJmxD/1218-terranova-12-data-atlas-vektordaten-integrieren)

File: projects/dashboards/data-atlas/src/app/accordion-vector-entry/accordion-vector-entry.component.ts [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        });
+    }
+
+    loadData(layerListing: LayerCollectionLayerDict): void {
+        const id = layerListing.id;
+
+        this.layerCollectionService
+            .getLayer(id.providerId, id.layerId)
+            .pipe(
+                mergeMap((layer) => combineLatest([of(layer), this.projectService.registerWorkflow(layer.workflow)])),
```
Comment (jdroenner) [inline projects/dashboards/data-atlas/src/app/accordion-vector-entry/accordion-vector-entry.component.ts:67]: you could use `registerWorkflowForLayer`from backend or `registerAndGetLayerWorkflowId` from the layer-collection service. But if you need the layer i guess it does not matter.

File: projects/dashboards/data-atlas/src/app/main/main.component.html [hunk 2]
```diff
@@ -10,7 +10,20 @@
...
 
-                <geoengine-accordion-entry [collection]="collection.id" [icon]="icon(collection.name)"></geoengine-accordion-entry>
+                <mat-tab-group dynamicHeight>
+                    <mat-tab label="Raster" *ngIf="collection.raster">
+                        <geoengine-accordion-entry
+                            [collection]="collection.raster.id"
+                            [icon]="icon(collection.name)"
+                        ></geoengine-accordion-entry>
+                    </mat-tab>
+                    <mat-tab label="Vector" *ngIf="collection.vector">
```
Comment (jdroenner) [inline projects/dashboards/data-atlas/src/app/main/main.component.html:20]: it is really strange that it does not show when there is no raster in the matching folder. Is it possible to change that?

File: projects/dashboards/data-atlas/src/app/accordion-vector-entry/accordion-vector-entry.component.ts (hunk 1 above)
Comment (ChristianBeilschmidt) [reply projects/dashboards/data-atlas/src/app/accordion-vector-entry/accordion-vector-entry.component.ts:67]: I need the layer for some metadata and also wrt. workflow metadata, I could not get rid of any step.

File: projects/dashboards/data-atlas/src/app/main/main.component.html (hunk 2 above)
Comment (ChristianBeilschmidt) [reply projects/dashboards/data-atlas/src/app/main/main.component.html:20]: [7770ee7](https://github.com/geo-engine/geoengine-ui/pull/611/commits/7770ee7cd15f5059818e6175ea48f9016eefa7b5)

File: projects/dashboards/data-atlas/src/app/accordion-vector-entry/accordion-vector-entry.component.ts (hunk 1 above)
Comment (ChristianBeilschmidt) [reply projects/dashboards/data-atlas/src/app/accordion-vector-entry/accordion-vector-entry.component.ts]: Now I got it:

[8c6e7d6](https://github.com/geo-engine/geoengine-ui/pull/611/commits/8c6e7d6ae9dfcbe968e4183a75a09a577784bf49)

**Review by jdroenner — APPROVED** [review]

**Comment by jdroenner** [comment]
bors merge

**Pull request merged**